docs/releases/5.2.txt

```diff
@@ -265,6 +265,9 @@ Models
   longer required to be set on SQLite, which supports unlimited ``VARCHAR``
   columns.
 
+* The new :attr:`QuerySet.executed_query <.QuerySet.executed_query>`
+  attribute returns the SQL query if the QuerySet has been evaluated.
```

Comment:
This only happens if the connection is actually building up a query log right? And so even if a queryset is evaluated there are times where `executed_query is None` right? Or am I misreading something about `Connection.queries`?